spec.md

```diff
@@ -62,6 +62,14 @@ be interpreted as described in [RFC 2119](https://tools.ietf.org/html/rfc2119).
 
 This specification defines the following terms:
 
+- *Application*: Often the entity using a Service Instance will be a piece of
+  software, however, this does not need to be the case. For the purposes of
+  this specification, the term "Application" will be used to represent all
+  entities that might make use of, and be bound to, a Service Instance.
+
+- *Marketplace*: The collective offerings brought by a Service Broker for a
```

Comment:
How about:

Marketplace: An aggregation of services provided by Service Brokers that may have been curated before presented to end users of a platform.

Reply:
I like the direction, I worked it in and updated.